src/sendtohelix.proj

```diff
@@ -33,11 +33,19 @@
     <!-- This property is used to show the tests results in Azure Dev Ops. By setting this property the
     test run name will be displayed as $(BuildConfiguration)-$(HelixTargetQueue) -->
     <TestRunNamePrefix>$(BuildConfiguration)-</TestRunNamePrefix>
-    <TestRunNamePrefix Condition="'$(IsPackageTesting)' == 'true'">PackageTests-$(ConfigurationGroup)-$(ArchGroup)</TestRunNamePrefix>
 
     <FailOnTestFailure Condition="'$(WaitForWorkItemCompletion)' != ''">$(WaitForWorkItemCompletion)</FailOnTestFailure>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetGroup)' == 'AllConfigurations'">
+    <!-- Use helix feature to include dotnet CLI for every workitem and add it to the path -->
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+
+    <GlobalJsonContent>$([System.IO.File]::ReadAllText('$(RepoRoot)global.json'))</GlobalJsonContent>
```

Comment:
Cool, all we have to do is set this?  That is super easy.  Thanks for figuring this out.

Reply:
Yes. I tested it by sending a private helix job and it worked like a charm:

This are the logs for a workitem:

```
C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>set DOTNET_CLI_TELEMETRY_OPTOUT=1 

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>set DOTNET_MULTILEVEL_LOOKUP=0 

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>set PATH=C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Payload\dotnet;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Users\packer\AppData\Local\Microsoft\WindowsApps;C:\Python;C:\Users\runner\AppData\Local\Microsoft\WindowsApps;C:\python3.7.0\lib\site-packages\pywin32_system32 

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>dotnet msbuild C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Payload\test.msbuild /warnaserror /p:PackageTestProjectsDir=C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Unzip /p:RestorePackagesPath=C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Unzip\packages /p:LocalPackagesPath="C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Payload\packages\" 
Microsoft (R) Build Engine version 16.3.0+0f4c62fea for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 5.82 sec for C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Unzip\netcoreapp5.0\project.csproj.
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=win7-x86
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=win7-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=centos.7-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=debian.8-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=linuxmint.17-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=rhel.7.2-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=ubuntu.14.04-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=ubuntu.16.04-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=osx.10.12-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=fedora.24-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=opensuse.42.1-x64
  Testing System.Runtime.Intrinsics.Experimental TFM=netcoreapp5.0 RID=rhel.7-x64

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>set _commandExitCode=0 

C:\dotnetbuild\work\3adb2841-ebac-4e65-8e5b-f586f209ca1e\Work\af2affd4-7980-4ef4-85b6-42adc1cb3686\Exec>EXIT /b 0 
```

You can see it is the version we have in `global.json` and that they add it to the path right before the execution 😄